README.md

```diff
@@ -1,20 +1,18 @@
-# ManageIQ::API::Common
+# Insights::API::Common
 
-[![Build Status](https://travis-ci.org/ManageIQ/manageiq-api-common.svg)](https://travis-ci.org/ManageIQ/manageiq-api-common)
-[![Maintainability](https://api.codeclimate.com/v1/badges/790ea6c77d82da6be68a/maintainability)](https://codeclimate.com/github/ManageIQ/manageiq-api-common/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/790ea6c77d82da6be68a/test_coverage)](https://codeclimate.com/github/ManageIQ/manageiq-api-common/test_coverage)
-[![Security](https://hakiri.io/github/ManageIQ/manageiq-api-common/master.svg)](https://hakiri.io/github/ManageIQ/manageiq-api-common/master)
+[![Build Status](https://travis-ci.org/RedHatInsights/insights-api-common-rails.svg)](https://travis-ci.org/RedHatInsights/insights-api-common-rails)
+[![Maintainability](https://api.codeclimate.com/v1/badges/790ea6c77d82da6be68a/maintainability)](https://codeclimate.com/github/RedHatInsights/insights-api-common-rails/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/790ea6c77d82da6be68a/test_coverage)](https://codeclimate.com/github/RedHatInsights/insights-api-common-rails/test_coverage)
+[![Security](https://hakiri.io/github/RedHatInsights/insights-api-common-rails/master.svg)](https://hakiri.io/github/RedHatInsights/insights-api-common-rails/master)
```

Comment:
https://hakiri.io/github/RedHatInsights/insights-api-common-rails/master 
this link is not working and there is similar issue on some topo repos. If I can make it work let me know or I will create issue/jira issue to track this issue somewhere.

Reply:
@lpichler I believe either myself or @gtanzillo can update the hakiri webhook on the repos.  I suggest opening an issue so we can work through updating the other repos.

Reply:
it looks that it is fixed for this repo, thanks 👍 

anyway I created separated issues for other repos where it needs to be fixed:
https://github.com/RedHatInsights/topological_inventory-api-client-ruby/issues/27
https://github.com/RedHatInsights/topological_inventory-ansible_tower/issues/68
https://github.com/RedHatInsights/topological_inventory-amazon/issues/57